CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# 1.4.2 (8 October 2021)
+
+* [+] Users can now automatically configure records for the domain without hosting. (EXTPLESK-2993)
+* [*] Updated templates.
+* [-] The authentication is no longer required to determine if a specific template is supported.
+* [-] Domain Connect no longer crashes while DNS records configuring. (EXTPLESK-2602)
```

Comment:
`* [-] The extension can no longer crash while configuring DNS records. (EXTPLESK-2602)`